lang/cs/Org.Apache.REEF.Client/YARN/IJobResourceUploader.cs

```diff
@@ -15,12 +15,13 @@
 // specific language governing permissions and limitations
 // under the License.
 
+using System.Collections.Generic;
 using Org.Apache.REEF.Tang.Annotations;
 
 namespace Org.Apache.REEF.Client.Yarn
 {
     [DefaultImplementation(typeof(LegacyJobResourceUploader))]
-    public interface IJobResourceUploader
+    internal interface IJobResourceUploader
```

Comment:
Initially this was made public on purpose to allow a user to provide custom implementation of job resource uploader. This is part of two ways to allow user to provide resource uploading capabilities. One via providing IFileSystem implementation and then using the FileSystemJobResouceUploader and another by implementing the IJobResourceUploader interface.
Is there a specific reason why we are changing this interface to internal?